server/api/controllers/admin.js

```diff
@@ -2114,6 +2114,44 @@ const getUserReferer = (req, res) => {
 		});
 };
 
+const createUserByAdmin = (req, res) => {
+	const { email, password } = req.swagger.params.data.value;
+
+	loggerAdmin.info(req.uuid, 'controllers/admin/createUserByAdmin email', email);
+
+	const userId = req.auth.sub.id;
+
+	toolsLib.user.getUserByKitId(userId)
```

Comment:
Why do you have to get the user id from the db to make sure he is admin? We already have the jwt token that passed the auth with admin scope @fetok12 ?